manifest.json

```diff
@@ -12,7 +12,9 @@
         "default_icon": "resources/vibinex-logo.png"
     },
     "permissions": [
-        "storage"
+        "storage",
+        "https://vibinex.com/**",
+        "cookies"
```

Comment:
Both are part of the "cookies" permission. All urls for which cookies are allowed need to be mentioned here